### DISCUSSION TITLE
Do the developers follow the forum (forum.transmissionbt.com)?

### DISCUSSION THREAD

**Discussion by ghost** [transmission/transmission#2648] [General]
There's a link on the homepage ("support"). [I've posted a handful of GTK questions](https://forum.transmissionbt.com/search.php?author_id=40151&sr=posts) (with some research behind them) and despite views between ten and twenty thousand there are no replies by other users.

---

I only recently became aware that the Github discussions section was opened — I'll migrate some here soon.

**Comment by ckerr** [comment]
- The maintainers mostly do not follow the forum; it's more for user-to-user support.

- Bug reports should be filed at https://github.com/transmission/transmission/issues and are regularly triaged by maintainers.

Please note that this is a very small project and the maintainers' time is stretched (very) thin. The project has had a bad reputation for being nonresponsive in the past.  While I'm trying to improve that, but it's not going to scale well if people (not just you) start asking lots of support-style questions from maintainers. Please try user-to-user support first :heart:
